.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "thirdparty/webview"]
 	path = thirdparty/webview
-	url = https://github.com/iwillspeak/webview.git
-	branch = webview-opaque
+	url = https://github.com/zenakuten/webview.git
```

Comment:
You probably want to ignore this change.  I wanted my fork to work out of the box.